skbio/io/format/genbank.py

```diff
@@ -0,0 +1,865 @@
+
++------+------+---------------------------------------------------------------+
+|Reader|Writer|                          Object Class                         |
++======+======+===============================================================+
+|Yes   |Yes   |:mod:`skbio.sequence.Sequence`                                 |
++------+------+---------------------------------------------------------------+
+|Yes   |Yes   |:mod:`skbio.sequence.DNA`                                      |
++------+------+---------------------------------------------------------------+
+|Yes   |Yes   |:mod:`skbio.sequence.RNA`                                      |
++------+------+---------------------------------------------------------------+
+|Yes   |Yes   |:mod:`skbio.sequence.Protein`                                  |
++------+------+---------------------------------------------------------------+
+|Yes   | Yes  | generator of :mod:`skbio.sequence.Sequence` objects           |
++------+------+---------------------------------------------------------------+
+
+Format Specification
+--------------------
+**State: Experimental as of 0.4.0-dev.**
+
+The International Nucleotide Sequence Database Collaboration (INSDC)
+foundational initiative between the DDBJ, EMBL, and GenBank
+(http://www.insdc.org/). These organisations all use the
+same "Feature Table" layout in their plain text flat file formats.
+
+However, the header and sequence sections of an EMBL file are very
+different in layout to those produced by GenBank/DDBJ.
+
+Feature Table Documentation:
+http://www.insdc.org/files/feature_table.html
+ftp://ftp.ncbi.nih.gov/genbank/docs/FTv10_3.html
+
+The sequence in the ``'ORIGIN'`` section is always in lowercase for the
+GenBank files downloaded from NCBI. For the RNA molecules, ``'t'`` (thymine),
+instead of ``'u'`` (uracil) is used in the sequence. All GenBank writers
+follow these traditions while writing GenBank files.
+
+Format Parameters
+-----------------
+
+Reader-specific Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+The ``constructor`` parameter can be used with the ``Sequence`` generator
+to specify the in-memory type of each GenBank record that is parsed.
+``constructor`` should be ``Sequence`` or a sub-class of ``Sequence``.
+It is also detected by the unit label on the LOCUS line. For example, if it
+is ``'bp'``, it will be read into ``DNA``; if it is ``'aa'``, it will be read
+into ``Protein``. Otherwise, it will be read into ``Sequence``. This default
+behavior is overridden by setting ``constructor``.
+
+``lowercase`` is another parameter available for all GenBank readers.
+By default, it is set to ``True`` to read in the ``'ORIGIN'`` sequence
+as lowercase letters. This parameter is passed to ``Sequence`` or
+its sub-class constructor.
+
+``seq_num`` is a parameter used with the ``Sequence``, ``DNA``, ``RNA``, and
+``Protein`` GenBank readers. It specifies which GenBank record to read from the
+concatenated GenBank file with multiple records in it.
```

Comment:
`It specifies which GenBank record to read from the concatenated GenBank file with multiple records in it.` -> `It specifies which GenBank record to read from a GenBank file with multiple records in it.`